index.html

```diff
@@ -52,45 +52,43 @@ <h2 class="hero__description">Subscribe today and get notified first when we lau
         <div class="container">
             <div class="offer__content">
                 <div class="offer__table">
-                    <div class="offer__collumn">
+                    <div class="offer__column">
                         <div class="offer__pic">
-                            <img src="./src/assets/icon/illustration1.svg" class="offer__img1" alt="ikona procentów">
+                            <img src="./src/assets/icon/illustration1.svg" class="offer__img%" alt="icon of percentage">

```

Comment:
avoid using symbols in class names, this `%` sign looks like a bug 😒

Reply:
done